src/hooks/useLevels.ts

```diff
@@ -17,20 +17,13 @@ type LevelsReturnType = {
 
 export function useLevels({ gameId, address }: Props) {
   const { data, isLoading, error } = useQuery<LevelsReturnType>(
-    ['levels', gameId],
+    ['levels'],
```

Comment:
gameId won't change so this is fine but I think it's best practice to have descriptive cache key as much as possible

Reply:
removed to avoid additional changes to the hooks i/o since we don't need